src/taskpane/server.js

```diff
@@ -1135,40 +1335,70 @@ function contentValidationSetter(sheet, model, fieldTypeEnums, context) {
         break;
     }
   }
+  if (!IS_GOOGLE) {
+    return context.sync();
+  }
 }
 
 
-
 // create dropdown validation on set column based on specified values
 // @param: sheet - the sheet object
 // @param: columnNumber - int of the column to validate
 // @param: rowLength - int of the number of rows for range (global param)
 // @param: list - array of values to show in a dropdown and use for validation
 // @param: allowInvalid - bool to state whether to restrict any values to those in validation or not
-function setDropdownValidation(sheet, columnNumber, rowLength, list, allowInvalid) {
+async function setDropdownValidation(mainSheetName, columnNumber, list, allowInvalid, context, model) {
   if (IS_GOOGLE) {
-    // create range
-    var range = sheet.getRange(2, columnNumber, rowLength);
+    //first, write the values to the dbSheet
+    var dataBaseSheetName = params.dataSheetName + model.currentProject.id + model.currentArtifact.id;

```

Comment:
Given that the dbSheetName is needed in a number of places, consider making a function for it - that way you can change the format in a single place. For example:
```
function createDatabaseSheetName(sheetName, projectId, artifactTypeId) {
   return sheetName + "-" + projectId + "-" + artifactTypeId;
}
```

Reply:
Good idea! Changed.